### PR TITLE
HLF v1.2.1

### PR DESCRIPTION
**Pull request by Dviejopomata**
New version of kubectl-hlf for https://github.com/kfsoftware/hlf-operator

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/hlf.yaml` is not a straightforward version bump: change on line `6` does not seem like a version bump: [`-  shortDescription: "Deploy and manage Hyperledger Fabric components"`]

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/hlf.yaml` is not a straightforward version bump: change on line `6` does not seem like a version bump: [`-  shortDescription: "Deploy and manage Hyperledger Fabric components"`]

File: plugins/hlf.yaml [hunk 1]
```diff
@@ -3,13 +3,12 @@ kind: Plugin
 metadata:
   name: hlf
 spec:
-  version: "v1.1.0"
+  version: "v1.2.1"
   homepage: https://github.com/kfsoftware/hlf-operator
-  shortDescription: "Deploy and manage Hyperledger Fabric components"
+  shortDescription: "Deploy and manage Hyperledger Fabric Peers, Orderers and Certificate authorities"
```
Comment (ahmetb) [inline plugins/hlf.yaml:8]: This needs to be short and fit to krew list

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/hlf.yaml` is not a straightforward version bump: change on line `8` does not seem like a version bump: [`-    The kubectl-hlf plugin wraps the HLF Operator and provides a simple`]

File: plugins/hlf.yaml [hunk 2]
```diff
@@ -3,13 +3,12 @@ kind: Plugin
...
-  version: "v1.1.0"
+  version: "v1.2.1"
   homepage: https://github.com/kfsoftware/hlf-operator
   shortDescription: "Deploy and manage Hyperledger Fabric components"
   description: |
-    The kubectl-hlf plugin wraps the HLF Operator and provides a simple
-    interface to create and manage Hyperledger Fabric Peers, 
-    Orderers and Certificate authorities
+    The kubectl-hlf plugin wraps the Hyperledger Fabric Operator and provides a simple
+    interface to create and manage Hyperledger Fabric Peers, Orderers and Certificate authorities
```
Comment (ahmetb) [inline plugins/hlf.yaml:11]: These were line wrapped at 80. Please restore.

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/hlf.yaml` is not a straightforward version bump: change on line `7` does not seem like a version bump: [`+      files:`]

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/hlf.yaml` is not a straightforward version bump: change on line `6` does not seem like a version bump: [`-  shortDescription: "Deploy and manage Hyperledger Fabric components"`]

File: plugins/hlf.yaml [hunk 3]
```diff
@@ -19,20 +18,36 @@ spec:
...
     - selector:
         matchLabels:
           os: linux
           arch: amd64
-      uri: https://github.com/kfsoftware/hlf-operator/releases/download/v1.1.0/kubectl-hlf_linux_amd64.zip
-      sha256: f586be1e5c4de89576c8953b65af61be4a6defa1b4d0ede29d31b2bd823b4647
+      uri: https://github.com/kfsoftware/hlf-operator/releases/download/v1.2.1/kubectl-hlf_1.2.1_linux_amd64.zip
+      sha256: 80ff54d8014ec884be444afc471e05f43abb7fe50580ac07cd1f768cfd457455
+      files:
+        - from: kubectl-hlf_1.2.1_linux_amd64
```
Comment (ahmetb) [inline plugins/hlf.yaml:37]: This file name will cause auto-approvals to fail at the next version. No other plugin's binary names in the archive contain version strings.

Why do you list files explicitly here? Are there other stuff in the archive?

**Review by ahmetb — CHANGES_REQUESTED** [review]

File: plugins/hlf.yaml (hunk 3 above)
Comment (Dviejopomata) [reply plugins/hlf.yaml:37]: If I remove the files will it work?

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/hlf.yaml` is not a straightforward version bump: change on line `6` does not seem like a version bump: [`-  shortDescription: "Deploy and manage Hyperledger Fabric components"`]

File: plugins/hlf.yaml (hunk 3 above)
Comment (Dviejopomata) [reply plugins/hlf.yaml]: I have removed the files section and re-uploaded the zips with the right binary names.

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/hlf.yaml` is not a straightforward version bump: change on line `23` does not seem like a version bump: [`-      bin: kubectl-hlf.exe`]

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/hlf.yaml` is not a straightforward version bump: changing old version (`"1.1.0"`) with new version (`"1.2.1"`) in the url (`https://github.com/kfsoftware/hlf-operator/releases/download/v1.1.0/kubectl-hlf_darwin_amd64.zip`) does not appear in the patch as `https://github.com/kfsoftware/hlf-operator/releases/download/v1.2.1/kubectl-hlf_darwin_amd64.zip`

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request seems to be a straightforward version bump. I'll go ahead and accept it. :+1: Cheers.

/lgtm
/approve

File: plugins/hlf.yaml (hunk 1 above)
Comment (Dviejopomata) [reply plugins/hlf.yaml]: Solved

File: plugins/hlf.yaml (hunk 2 above)
Comment (Dviejopomata) [reply plugins/hlf.yaml]: Solved

**Pull request merged**